index.html

```diff
@@ -3243,15 +3241,34 @@ <h3><dfn>Writing content</dfn></h3>
         To <dfn>map external data to NDEF</dfn> given |record:NDEFRecordInit|,
         |ndef|, and |recordsDepth:unsigned short|, run these steps:
         <ol class=algorithm data-link-for="NDEFRecordInit">
-          <!--
           <li>
-            If |context| is `"smart-poster"`,
+            If |record|'s <a>mediaType</a> is not `undefined`,
             [= exception/throw =] a {{TypeError}} and abort these steps.
           </li>
-          -->
           <li>
-            If |record|'s <a>mediaType</a> is not `undefined`,
-            [= exception/throw =] a {{TypeError}} and abort these steps.
+            Let |domain| and |type| be the result of running [=split external type=] with
+            |record|'s <a>recordType</a>.
+          </li>
+          <aside class="note">
+            The above step is ensures to work due to earlier validation of the external type.
```

Comment:
Nit: "The above step is ensured to..." or "The above step always succeeds due to..."